core/iwasm/common/wasm_runtime_common.c

```diff
@@ -1794,6 +1798,9 @@ wasm_runtime_call_wasm(WASMExecEnv *exec_env,
                                              result_argc, argv)) {
         wasm_runtime_set_exception(exec_env->module_inst,
                                    "the result conversion is failed");
+#if WASM_ENABLE_THREAD_MGR != 0
+        wasm_cluster_spread_exception(exec_env);
```

Comment:
Should also spread exception when wasm_runtime_prepare_call_function fails, L1767?

Reply:
according to our discussion, I‘ve moved the spread into wasm_set_exception function, so we don't need to process the spread elsewhere